run.sh

```diff
@@ -1,7 +1,10 @@
 #!/usr/bin/env bash
 
+
+
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 echo "skycoin binary dir:" "$DIR"
 pushd "$DIR" >/dev/null
+
```

Comment:
Whitespaces should not pollute the PR diff. Make sure this file does not get listed .